### DISCUSSION TITLE
ZLUDA failed to initialize: no HIP SDK found

### DISCUSSION THREAD

**Discussion by LilJew** [vladmandic/sdnext#3315] [Q&A]
Hello Guys. I tried to install ZLUDA,because Direct ML generation speed is too slow,but executable file of SDNext shows me a sad message,
![изображение](https://github.com/vladmandic/automatic/assets/79800122/652df11b-8217-4f29-94e1-a6985f1c24a0)
I installed this SDK and checked Enviroment Variables, all according to tutorial, any ideas? My card btw is AMD RX6600XT Thanks in advance

**Comment by brknsoul** [comment]
Did you install HIP SDK from https://www.amd.com/en/developer/resources/rocm-hub/hip-sdk.html ?

Ensure you're following the guide; https://github.com/vladmandic/automatic/wiki/ZLUDA

If you're not installing into the default place (Program Files), then you'll need to add a HIP_PATH environment variable pointing to where you've installed it.

**Reply by LilJew** [reply]
Yes, I install this, and in my environment variables existing two versions of HIP_PATH variables,but I still get the warning message, screen of my Environment Variables below: 
![изображение](https://github.com/vladmandic/automatic/assets/79800122/4bd6c5b2-4d6b-44fc-a88c-c1a4c37af5c2)
Second screenshot is my PATH variable:
![изображение](https://github.com/vladmandic/automatic/assets/79800122/60c02760-32c4-4870-9d32-b2845770be83)
Should I add something to this maybe?

**Reply by brknsoul** [reply]
- What GPU do you have?
- Do you have an AMD integrated GPU on your CPU (iGPU)?
- Where is SD.Next installed?

**Reply by LilJew** [reply]
I have  an AMD RX6600xt 8GB GPU
No, I didn't have iGPU
SD.Next installed in C:\Users\kirill\automatic

**Reply by ns-rosa** [reply]
I have a similar issue. Did you find a way to fix it?

**Reply by brknsoul** [reply]
Please don't comment on closed topics. ZLUDA isn't supported via Issues, but by the ZLUDA Questions and Support thread in the help channel on SD.Next Discord.

**Comment by MechaDeka2** [comment]
Why the actual hell would you wall troubleshooting information inside a Discord server? Do you realize this thread is the FIRST THING that comes up when googling this exact issue? Literally what do you gain out of obfuscating information inside a chat client with an absolutely useless search function?

**Reply by vladmandic** [reply]
tone it down and watch the language.
and it was pointed to discord at that time since zluda was highly experiemental and unstable development at that time and we needed direct interaction - thus the discord.
